.gitignore

```diff
@@ -130,13 +129,9 @@ ios/celo/Base.lproj/*
 ios/celo/Images.xcassets/
 ios/.xcode.env
 src/brandingConfig.ts
-src/icons/LogoHeart.tsx
-src/icons/Logo.tsx
-src/icons/WelcomeLogo.tsx
-src/icons/Home.tsx
+src/brandedIcons/*
 src/images/*
 !src/images/Images.ts
-!src/images/link-arrow.png
```

Comment:
should we delete `link-arrow.png` from the repo?

Reply:
yes it's been removed!